### PR TITLE
Async loading of world

### PR DESCRIPTION
**Pull request by JaXt0r**
# To test
1. Start game
2. Look how performance (frames) are when loading screen is there
3. Check performance once loading is done and world is availably
4. Move to another world (e.g. oldmine)
5. Move back to main world
6. Does everything work? Performance is fine? And you can still walk and grab things?

# Checklist
(Please judge for yourself which entry is valuable for your PR)

## Scene
* [x] FeatureFlags reverted to right value

# Testing
* [ ] Merged _main_ into this branch and tested with the latest features
* [x] Tested with PCVR
* [ ] Tested with Pico / Quest

### CODE REVIEW

**Review by JaXt0r — COMMENTED** [review]
I'm good with the PR. Checked our code base and tested it on PCVR.
@JucanAndreiDaniel please check once more on your end.
@Romeeey will test on Q2.

Then we can finally merge. 🕺

File: Assets/GothicVR/Scripts/Creator/WorldCreator.cs [hunk 1]
```diff
@@ -42,10 +40,12 @@ public void Create(string worldName)
 
             if (FeatureFlags.I.CreateOcNpcs)
                 PxVm.CallFunction(GameData.I.VmGothicPtr, "STARTUP_OLDCAMP");
-            
-            SceneManager.MoveGameObjectToScene(worldGo, SceneManager.GetSceneByName(worldName));
+
```
Comment (JucanAndreiDaniel) [inline Assets/GothicVR/Scripts/Creator/WorldCreator.cs:43]: Add back the world gameobject move to scene

```suggestion
            GvrSceneManager.I.MoveToWorldScene(worldGo);
```

**Review by JucanAndreiDaniel — CHANGES_REQUESTED** [review]
Just one small change

File: Assets/GothicVR/Scripts/Creator/WorldCreator.cs (hunk 1 above)
Comment (JucanAndreiDaniel) [reply Assets/GothicVR/Scripts/Creator/WorldCreator.cs:43]: this is not the proper fix :D

File: Assets/GothicVR/Scripts/Manager/GvrSceneManager.cs [hunk 2]
```diff
@@ -14,93 +17,161 @@ public class GvrSceneManager : SingletonBehaviour<GvrSceneManager>
...
+        private void HideLoadingScene()
         {
+            SceneManager.UnloadSceneAsync("Loading");
+
+            LoadingManager.I.ResetProgress();
+        }
+        
+        private void OnWorldSceneLoaded(Scene scene, LoadSceneMode mode)
+        {
+            if (scene.name == newWorldName)
```
Comment (JucanAndreiDaniel) [inline Assets/GothicVR/Scripts/Manager/GvrSceneManager.cs:125]: our scene is lower case but newWorldName might be all uppercase
```suggestion
            if (scene.name.toLower() == newWorldName.toLower())
```

File: Assets/GothicVR/Scripts/Creator/WorldCreator.cs (hunk 1 above)
Comment (JaXt0r) [reply Assets/GothicVR/Scripts/Creator/WorldCreator.cs:43]: Fixed bug otherwise

**Review by JucanAndreiDaniel — APPROVED** [review]
Fine for me now

**Pull request merged**